vars/dockerBuildNodeComponent.groovy

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/groovy
 
-def call(name, dockerhubOrg="feedhenry", dockerHubRepo=name, credentialId="dockerhubjenkins") {
-  def pkgVersion = sh(returnStdout: true, script: "node -p -e \"require('./package.json').version\"").trim().split('-')[0]
-  dockerBinaryBuild(name,dockerhubOrg,dockerHubRepo,credentialId,pkgVersion) {
+def call(name, dockerHubOrg = "feedhenry", dockerHubRepo = name, credentialId = "dockerhubjenkins") {
+    def version = sh(returnStdout: true, script: "node -p -e \"require('./package.json').version\"").trim().split('-')[0]
     sh "cp ./dist/fh-*x64.tar.gz docker/"
-    sh "oc start-build ${name}-bc --follow=true --from-dir ./docker" 
-  }
-}
\ No newline at end of file
+    dockerBinaryBuild(name, version, dockerHubOrg, dockerHubRepo, credentialId, './docker')
```

Comment:
Optional: I think you can remove the final `'./docker'` param from here if you want, since it's the default anyway